data/curated/new_submission/meta.yaml

```
@@ -0,0 +1,16 @@
+title: "What have we been watching on Netflix?"
+article:
+  title: "What We Watched the First Half of 2025"
+  url: "https://about.netflix.com/en/news/what-we-watched-the-second-half-of-2023", "https://about.netflix.com/en/news/what-we-watched-the-first-half-of-2024",
+"https://about.netflix.com/en/news/what-we-watched-the-second-half-of-2024", "https://about.netflix.com/en/news/what-we-watched-the-first-half-of-2025"
```

```suggestion
  url: "https://about.netflix.com/en/news/what-we-watched-the-second-half-of-2023, https://about.netflix.com/en/news/what-we-watched-the-first-half-of-2024, https://about.netflix.com/en/news/what-we-watched-the-second-half-of-2024, https://about.netflix.com/en/news/what-we-watched-the-first-half-of-2025"
```
Combine into a single string containing a comma-separated list, for automation.